pmda/density.py

```diff
@@ -0,0 +1,222 @@
+import numpy as np
+import MDAnalysis as mda
+import time
+from MDAnalysis.lib.util import fixedwidth_bins
+from MDAnalysis.analysis.density import Density
+from .parallel import ParallelAnalysisBase
+
+class pDensity(ParallelAnalysisBase):
```

Comment:
come up with a better name
- start with capital letter (PEP 8?)
- DensityCreator, DensityAnalysis, ... ????? Should be similar to what's in MDA (anythin based on AnalysisBase)